php/src/Communicator.cpp

```diff
@@ -835,7 +835,7 @@ ZEND_METHOD(Ice_ValueFactoryManager, add)
     ValueFactoryManagerPtr _this = Wrapper<ValueFactoryManagerPtr>::value(getThis());
     assert(_this);
 
-    zend_class_entry* factoryClass = idToClass("Ice::ValueFactory");
```

Comment:
Is there some special PHP reason not to have a leading `\` here?
Everywhere else, we had a leading slash.

Reply:
I don't think so. We should us `\\Ice`

Reply:
Yes, I think that we do overly qualify things, but for now, I'm just going to add it here to be consistent.
Instead of fixing all of our over-qualifications! : vP

Reply:
In PHP the global scope is important in some place, depending of how types are included.

For example I remember we have issues when you dynamically require Ice files inside a PHP function, and some PHP frameworks do that.